CHANGE-NOTES.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## v4.0.0
```

Comment:
Nit: let's keep it v3.8.0 (or even v3.7.1, no new feature theoretically 🤓 ) for now